### PR TITLE
PDR-4532 Updated Task Mining TMAnalyzerModel package to 1.1.6

### PR DESCRIPTION
**Pull request by danyloid**
- TM model package is available here: https://uipathdownloadstaging.blob.core.windows.net/publicmodels/AIC/TaskMining/21/task_mining.zip
- Updated to manifest file to only reference the latest version of the model, we don't need to reference to 1.1.5 (v2 previously)
- The original PR: https://github.com/UiPath/ai-customer-scripts/pull/181
- Jira ticket: https://uipath.atlassian.net/browse/PDR-4532
- This change is meant for AS 22.4 as a hotfix to the data issue

### CODE REVIEW

File: metadata/TMAnalyzerModel__21__metadata.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON38",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Task Mining",
+    "projectDescription": "UiPath Task Mining models to analyze captured data",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v21.10",
+    "languageVersion": 0,
+    "version": 21,
```
Comment (mohitseth-nitrr) [inline metadata/TMAnalyzerModel__21__metadata.json:22]: @vivekjhauipath @vpn87  Can you guys confirm if this kind of jump is ok to have w.r.t OOB installer.

Comment (danyloid) [reply metadata/TMAnalyzerModel__21__metadata.json:22]: Please let me know if that is not recommended - will change the version to `2`

Comment (vivekjhauipath) [reply metadata/TMAnalyzerModel__21__metadata.json:22]: It will work, aesthetics I can't comment on

**Review by aakashuipath — APPROVED** [review]
lgtm

**Pull request merged**